packages/dx-react-chart/docs/guides/animation.md

```diff
@@ -1,5 +1,19 @@
 # React Chart - Animation
 
-You can add the [Animation](../reference/animation.md) plugin to the Chart component to animate all the series when they first appear on-screen. To modify animation options, implement a custom point component for a series. In the following example, the duration of each bar's animation is configured individually:
+You can add the [Animation](../reference/animation.md) plugin to the Chart component to animate all the series.
 
-.embedded-demo({ "path": "chart-basic/animation", "showThemeSelector": true })
+## Basic Setup
+
+Import the `Animation` plugin.
```

Comment:
Can we add the `Related Plugins` section and one more demo to demonstrate Basic setup?

Also `Animation setups` -> `Animation Customization` or something else.

Reply:
We have not `Related Plugins`.